package.json

```diff
@@ -9,7 +9,7 @@
     "test": "jest"
   },
   "devDependencies": {
-    "@nrwl/nx-darwin-x64": "^15.9.3",
+    "nx": "^16.9.1",
```

Comment:
oh is it okay that we're going up to vs 16 from 15?

Reply:
Yeah ... I think it is ok. 
The newest version of @nrwl/nx-darwin-x64 is 15.9.7 and`nx` also depends on `15.9.7` version of @nrwl/nx-darwin-x64. Meaning the @nrwl/nx-darwin-x64 15.9.7 will be also installed when installing `nx`
I tested the command that we are using often (build/ publish). They all work fine.
cc: @juni-notifi

Reply:
I think @nrwl/nx-darwin-x64 version is not related to nx main version. It is difficult to fig out their dependency chain. 
But I realized when I install `nx` and `@nrwl/nx-darwin-x64` v15 and `@nx/nx-darwin-x64` v16 are both installed. 
And surprisingly they are both sharing the same content (`nx.darwin-x64.node`).
I tried all our often used commands (build/ publish). They both work fine with `nx` v16.